tests/test_installation.py

```diff
@@ -183,5 +189,79 @@ def test_pico_migrate(self):
             'city.yml', 'graph.xml', 'namespaces.txt']))
 
 
+class PicoModule(unittest.TestCase):
+    """Test the use of pico as a Python module."""
+
+    def setUp(self) -> None:
+        """Change installation path and reset the namespace registry."""
+        self._previous_installation_path = OntologyInstallationManager\
+            .get_default_installation_path()
```

Comment:
Shouldn't you get the `path` here, in case there was a custom one?

Reply:
When people use `osp.core.pico.install` and `osp.core.pico.uninstall`, what they expect is that the command acts on the default installation path. This is therefore the functionality I want to test here. This is also the reason why I added the `get_default_installation_path` and `set_default_installation_path` in the first place, just be able to change the path for the test. Otherwise the test would overwrite the user's ontologies. 

Non-default paths are already tested with the [other test case in the same file](https://github.com/simphony/osp-core/blob/9fbfe4dee86dfbf4494593e0cdc5cfd5194dab2e/tests/test_installation.py#L34).

Reply:
> When people use `osp.core.pico.install` and `osp.core.pico.uninstall`, what they expect is that the command acts on the default installation path

Are we sure of this?

Reply:
> > When people use `osp.core.pico.install` and `osp.core.pico.uninstall`, what they expect is that the command acts on the default installation path
> 
> Are we sure of this?

@yoavnash You asked for this feature, so please comment. From my side, I am sure of this.